plugins/commands/config/src/main/kotlin/ConfigCommand.kt

```diff
@@ -68,6 +69,7 @@ class ConfigCommand(descriptor: PluginDescriptor = ConfigCommandFactory.descript
 
     private val configWriter = YAMLMapper()
         .registerKotlinModule()
+        .setSerializationInclusion(JsonInclude.Include.NON_NULL)
```

Comment:
I'm not sure about that, isn't it better to show all config properties even if they are set to `null` to provide the whole picture?